quickcheck/tasty-quickcheck.cabal

```diff
@@ -25,7 +21,6 @@ Source-repository head
 
 library
   exposed-modules:     Test.Tasty.QuickCheck
-  -- other-modules:
   other-extensions:    GeneralizedNewtypeDeriving, DeriveDataTypeable
   build-depends:       base >= 4.8 && < 5,
```

Comment:
Would it make sense to lift `base` in this package to `>= 4.9`, since `tasty` has the same lower bound?

Reply:
Good idea, done.